app/themes/3column/ui/app/root/root.controller.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+  'use strict';
+
+  angular.module('app.root')
+    .controller('RootCtrl', RootCtrl);
+
+  RootCtrl.$inject = ['messageBoardService', 'userService', '$scope'];
+
+  function RootCtrl(messageBoardService, userService, $scope) {
+    var ctrl = this;
+    ctrl.currentYear = new Date().getUTCFullYear();
+    ctrl.messageBoardService = messageBoardService;
+
+    $scope.user = userService;
```

Comment:
I think we moved to putting such objects onto ctrl instead of $scope. Also not sure if root really needs this..